src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -101,6 +102,7 @@ public function __construct(array $options = array(), $handler = null, MetadataB
     {
         $options += array(
             'cache_limiter' => 'private_no_expire',
+            'cache_expire' => 0,
```

Comment:
Can't you then use cache_limiter => nocache directly. The only difference is that it also sets no-store, meaning no conditional requests.

Reply:
No-store is too aggressive to me, conditional requests are fine, aren't they?

Reply:
Should be yeah.